apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.Data/ConnectionPoolOptions.cs

```diff
@@ -0,0 +1,151 @@
+            set
+            {
+                switch (value)
+                {
+                    case LogLevel.Debug:
+                        Logger.LogLevel = V1.Logging.LogLevel.Debug;
+                        break;
+                    case LogLevel.Info:
+                        Logger.LogLevel = V1.Logging.LogLevel.Info;
+                        break;
+                    case LogLevel.Warn:
+                        Logger.LogLevel = V1.Logging.LogLevel.Warn;
+                        break;
+                    case LogLevel.Error:
+                        Logger.LogLevel = V1.Logging.LogLevel.Error;
+                        break;
+                    case LogLevel.None:
+                        Logger.LogLevel = V1.Logging.LogLevel.None;
+                        break;
+                    default:
+                        throw new ArgumentOutOfRangeException(nameof(value), value, null);
+                }
+            }
+        }
+
+        /// <summary>
+        /// </summary>
+        public bool LogPerformanceTraces
+        {
+            get { return Logger.LogPerformanceTraces; }
+            set { Logger.LogPerformanceTraces = value; }
+        }
+
+        /// <summary>
+        /// </summary>
+        public int MaximumActiveSessions
+        {
+            get { return SessionPool.MaximumActiveSessions; }
+            set { SessionPool.MaximumActiveSessions = value; }
+        }
+
+        /// <summary>
+        /// </summary>
+        public int MaximumPooledSessions
+        {
+            get { return SessionPool.MaximumPooledSessions; }
+            set { SessionPool.MaximumPooledSessions = value; }
+        }
+
+        /// <summary>
+        /// </summary>
+        public int PerformanceTraceLogInterval
+        {
+            get { return Logger.PerformanceTraceLogInterval; }
+            set { Logger.PerformanceTraceLogInterval = value; }
+        }
+
+        /// <summary>
+        /// </summary>
+        public TimeSpan PoolEvictTimeSpan
+        {
+            get { return SessionPool.PoolEvictTimeSpan; }
+            set { SessionPool.PoolEvictTimeSpan = value; }
+        }
+
+        /// <summary>
+        /// </summary>
+        public bool ResetPerformanceTracesEachInterval
+        {
+            get { return Logger.ResetPerformanceTracesEachInterval; }
+            set { Logger.ResetPerformanceTracesEachInterval = value; }
+        }
+
+        /// <summary>
+        /// </summary>
+        public ResourcesExhaustedBehavior ResourcesExhaustedBehavior
+        {
+            get
+            {
+                return SessionPool.WaitOnResourcesExhausted
+                    ? ResourcesExhaustedBehavior.Block
+                    : ResourcesExhaustedBehavior.Fail;
+            }
+            set { SessionPool.WaitOnResourcesExhausted = value == ResourcesExhaustedBehavior.Block; }
+        }
+
+        /// <summary>
+        /// </summary>
+        public int TimeoutMilliseconds { get; set; } = 600000;
```

Comment:
sort of a stretch to put in sessionpool, but it could be useful for initial session creation.  No user drawback, so sure.